src/AdminPage.tsx

```diff
@@ -47,8 +61,26 @@ const AdminChild = () => {
         </p>
       </>
     );
+
   }
 
+  const finalAdminEvent = eventAndKey.finalAdminEvent;
```

Comment:
I'm getting an error when submitting a form:
`finalAdminEvent is undefined
AdminChild@http://localhost:3000/static/js/bundle.js:482:91
renderWithHooks@http://localhost:3000/static/js/bundle.js:61178:31
mountIndeterminateComponent@http://localhost:3000/static/js/bundle.js:64464:17
beginWork@http://localhost:3000/static/js/bundle.js:65760:20
callCallback@http://localhost:3000/static/js/bundle.js:50770:18
invokeGuardedCallbackDev@http://localhost:3000/static/js/bundle.js:50814:20
invokeGuardedCallback@http://localhost:3000/static/js/bundle.js:50871:35
beginWork$1@http://localhost:3000/static/js/bundle.js:70745:32
performUnitOfWork@http://localhost:3000/static/js/bundle.js:69992:16
workLoopConcurrent@http://localhost:3000/static/js/bundle.js:69980:26
renderRootConcurrent@http://localhost:3000/static/js/bundle.js:69948:11
performConcurrentWorkOnRoot@http://localhost:3000/static/js/bundle.js:69282:42
workLoop@http://localhost:3000/static/js/bundle.js:83147:46
flushWork@http://localhost:3000/static/js/bundle.js:83125:18
performWorkUntilDeadline@http://localhost:3000/static/js/bundle.js:83362:25`

Reply:
I can't replicate this - which form did you submit?

Reply:
Just the standard form to create an event 😞 I can show you sometime on screen share?

Reply:
I made changes on this branch that resolved the type errors I was seeing; I still can't replicate your specific error but I think it's worth trying again. I also merged #50 into this so I could test easier; if you want to test this without installing Firebase CLI/testing PR #50 please comment out lines 30-32 on `/firebase/index.js`.